fastestimator/trace/io/__init__.py

```diff
@@ -20,4 +20,5 @@
 from fastestimator.trace.io.qms import QMSTest
 from fastestimator.trace.io.restore_wizard import RestoreWizard
 from fastestimator.trace.io.tensorboard import TensorBoard
+from fastestimator.trace.io.model_eval import ModelEval
```

Comment:
these are supposed to be sorted alphabetically

Reply:
fixed